libc/src/setjmp/CMakeLists.txt

```diff
@@ -2,6 +2,24 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${LIBC_TARGET_ARCHITECTURE})
   add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/${LIBC_TARGET_ARCHITECTURE})
 endif()
 
+if (LIBC_CONF_SETJMP_ENABLE_FORTIFICATION)
+  set(checksum_flags "-DLIBC_COPT_SETJMP_ENABLE_FORTIFICATION=1")
```

Comment:
Should we have a check to prevent this option being enabled in configurations where the functionality isn't supported? (Currently, both "not x86-64" and also "not Linux".)

I think if a user asks for fortification and can't have it, better to cause a build error than to silently build without the requested security feature.

Reply:
The way we provide these options make it hard to differentiate default values on different arches. I emit a warning in latest patch.